### ISSUE TITLE
Unable to extract proxy revision using Beta Version v2.0.0-beta.3

### ISSUE THREAD

**Issue by wrfleming** [apigee/apigeecli#403]
Version:  [v2.0.0-beta.3](https://github.com/apigee/apigeecli/releases/tag/v2.0.0-beta.3)

When using jq to extract proxy revision I'm getting a parsing error.   Works in current release yet not in beta version.  I have tried this with a couple different open spec files and get the same error.   If I remove the jq to retrieve the revision the proxy will not receive the error and the proxy is created.

<img width="1535" alt="image" src="https://github.com/apigee/apigeecli/assets/7915444/c99b2b57-df8f-404b-8f96-3d3d4b067919">

<img width="465" alt="image" src="https://github.com/apigee/apigeecli/assets/7915444/fc71b6dd-9e28-4793-8c6e-2d9600098692">

**Comment by kurtkanaskie** [comment]
I think that's due to a WARNING being printed to stdout instead of to stderr, which causes the response to be not valid JSON.

Change Line #54 in internal/clilog/log.go
from:
		warningHandle = os.Stdout
to:
		warningHandle = os.Stderr

**Comment by ssvaidyanathan** [comment]
Thanks @kurtkanaskie - Thats the exact problem

@wrfleming - I think that warning is useful and you will have to ignore it in your command to extract the revision.

Can you update your command to something like this?

```sh
REV=$($apigeepath/apigeecli apis create openapi -f apigeexpoc -n cloudrunhealthcheck --oas-name apidevlab.yml --target-server-name mocktest -t $(gcloud auth print-access-token | sed '1d' | jq ."revision" -r))
``` 

I tested it on my laptop and it worked

**Comment by ssvaidyanathan** [comment]
Fixed in v2.0.0-beta.4
